example-kotlin/src/main/java/com/attentive/example_kotlin/ExampleKotlinApp.kt

```diff
@@ -0,0 +1,34 @@
+package com.attentive.example_kotlin
+
+import android.app.Application
+import com.attentive.androidsdk.AttentiveConfig
+import com.attentive.androidsdk.AttentiveEventTracker
+import com.attentive.androidsdk.UserIdentifiers
+
+class ExampleKotlinApp : Application() {
+    lateinit var attentiveConfig: AttentiveConfig
+
+    companion object {
+        fun buildUserIdentifiers(): UserIdentifiers {
+            // Add all the identifiers that you have for the current user
+            return UserIdentifiers.Builder()
+                .withPhone("+15556667777")
+                .withEmail("some_email@gmailfake.com")
+                .withKlaviyoId("userKlaviyoId")
+                .withShopifyId("userShopifyId")
+                .withClientUserId("userClientUserId")
+                .withCustomIdentifiers(mapOf(Pair("customIdentifierKey", "customIdentifierValue")))
+                .build()
```

Comment:
we probably should add all the same comments (ie you only need to create the AttentiveConfig once, creating user identifiers is optional etc etc)  we have in the Java Example here so that clients don't have to look at both examples

Reply:
Our Java example app doesn't have these. I'll add them there too!